acquire/acquire.py

```diff
@@ -248,6 +248,49 @@ def _run(cls, target, collector):
         pass
 
 
+@register_module("--sys")
+@local_module
+class Sys(Module):
+    DESC = "Sysfs files (live systems only)"
+    EXEC_ORDER = ExecutionOrder.BOTTOM
+
+    @classmethod
+    def _run(cls, target: Target, collector: Collector):
+        if not Path("/sys").exists() or sys.platform == "win32":
```

Comment:
```suggestion
        if not Path("/sys").exists():
```
Just guessing here that the check for `win32` doesn't work, because `/sys` will never exist on Windows so that check will never be done.

Reply:
The recent change doesn't address this.